### PR TITLE
API: bump max size of teal source and dryrun

### PR DESCRIPTION
**Pull request by barnjamin**
addresses #4632

### CODE REVIEW

**Review by jannotti — DISMISSED** [review]
Seems totally reasonable, and this has been requested from external contributors..

File: daemon/algod/api/server/v2/handlers.go [hunk 1]
```diff
@@ -51,8 +51,14 @@ import (
 	"github.com/algorand/go-codec/codec"
 )
 
-const maxTealSourceBytes = 1e5
-const maxTealDryrunBytes = 1e5
+// max compiled teal program is currently 8k
+// but we allow for comments, spacing, and repeated consts
+// in the source teal, allow up to 200kb
+const maxTealSourceBytes = 2e5
```
Comment (jannotti) [inline daemon/algod/api/server/v2/handlers.go:57]: Now that know `_` is allowed in int constants in Go, I would prefer `200_000` here

Comment (barnjamin) [inline daemon/algod/api/server/v2/handlers.go:57]:
```suggestion
const maxTealSourceBytes = 200_000
```

File: daemon/algod/api/server/v2/handlers.go [hunk 2]
```diff
@@ -51,8 +51,14 @@ import (
...
-const maxTealSourceBytes = 1e5
-const maxTealDryrunBytes = 1e5
+// max compiled teal program is currently 8k
+// but we allow for comments, spacing, and repeated consts
+// in the source teal, allow up to 200kb
+const maxTealSourceBytes = 200_000
+
+// With the ability to hold unlimited assets DryrunRequests can
+// become quite large, allow up to 1mb
+const maxTealDryrunBytes = 1e6
```
Comment (barnjamin) [inline daemon/algod/api/server/v2/handlers.go:61]:
```suggestion
const maxTealDryrunBytes = 1_000_000
```

**Review by jannotti — APPROVED** [review]

**Pull request merged**